app/graphql/mutations/batch_export/create_batch_export.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Mutations::BatchExport::CreateBatchExport < Mutations::BaseMutation
+  argument :search_params, String, required: true
+
+  field :batch_export, Types::BatchExportType, null: true
+
+  def resolve(**attributes)
+    # Note: No ability-based authorization for creating a CSV export.
```

Comment:
I would honestly still do the ability check here. I know it will pass for all users, but if it changes in the future it offers us protection. We currently do the check for mutations like create new term even though everybody can create a new term.

Reply:
Good idea.  Will do!